cybersyn/scripts/central-planning.lua

```diff
@@ -149,16 +179,17 @@ function create_manifest(map_data, r_station_id, p_station_id, train_id, primary
 		local item_name = v.signal.name
 		local item_type = v.signal.type
 		local r_item_count = v.count
-		local r_effective_item_count = r_item_count + (r_station.deliveries[item_name] or 0)
+		local r_effective_item_count = r_item_count + (r_station.deliveries[item_hash] or 0)
```

Comment:
item_hash is not a local variable here

Reply:
Fixed in 1e2819aed231d1a5a998be2c05ae00c5ada00e9d